src/unitxt/metric_utils.py

```diff
@@ -320,6 +320,30 @@ def prepare(self):
 )
 
 
+class EvaluationResults(list):
+    @property
+    def score(self):
+        return self[0]["score"]["global"]
+
+    @property
+    def groups(self):
+        if "groups" not in self[0]["score"]:
+            raise ValueError("Groups scores not found try using group_by in the recipe")
+        return self[0]["score"]["groups"]
+
+    @property
+    def subsets(self):
+        if "subsets" not in self[0]["score"]:
+            raise ValueError("Subsets scores not found try using Benchmark")
+        return self[0]["score"]["subsets"]
+
+    def to_df(self):
+        import pandas as pd
+
+        # Flatten and load into DataFrame
+        return pd.json_normalize(self)
```

Comment:
How does the output look like? Maybe since it's a one line, we don't need it.  As it might make sense to convert different scores (global, subsets) seperately,